### PR TITLE
test(e2e): User group SPA additional test cases(invalid search,notification message, breadcrumb,documentation link redirection)

### PR DESCRIPTION
**Pull request by mudit94**
# Closes/Fixes/Resolves

### 

# Explain the feature/fix

test(e2e): User group SPA additional test cases(invalid search,notification message, breadcrumb,documentation link redirection)
## Does this PR introduce a breaking change

No

## Screenshots

<details>
<summary>View Screenshots</summary>

image

</details>

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [x] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demoed and the design review approved?

### CODE REVIEW

**Review by riginoommen — APPROVED** [review]

File: e2e/integration/usergrouptests.js [hunk 1]
```diff
@@ -23,13 +26,28 @@ context( 'Test for UserGroup', () => {
         } )
     })
 
-    it( 'Test for search user group by name', () => {
+    it( 'Test for search user group by rover group', () => {
         cy.get( '#searchText' ).type( 'dsal-admin' )
         cy.get( 'tbody' ).within( () => {
             cy.get( '[data-label^="LDAP"]' ).first().should( 'have.text','dsal-admin')
```
Comment (hybridx) [inline e2e/integration/usergrouptests.js:32]: @mudit94 why is there dsal-admin mentioned here?

**Review by hybridx — APPROVED** [review]

File: e2e/integration/usergrouptests.js (hunk 1 above)
Comment (mudit94) [reply e2e/integration/usergrouptests.js:32]: basically it is trying to search that group. in the list

Comment (mudit94) [reply e2e/integration/usergrouptests.js:32]: @hybridx

Comment (hybridx) [reply e2e/integration/usergrouptests.js:32]: @mudit94 can you change it to something else please?

**Review by deshmukhmayur — APPROVED** [review]
LGTM.

**Review by hybridx — APPROVED** [review]

**Pull request merged**